components/alibi-detect-server/adserver/cd_model.py

```diff
@@ -94,8 +97,10 @@ def process_event(self, inputs: Union[List, Dict], headers: Dict) -> Optional[Mo
                 self.batch.shape[0],
                 self.drift_batch_size,
             )
-
-            cd_preds = self.model.predict(self.batch)
+            if DRIFT_TYPE_FEATURE:
+                cd_preds = self.model.predict(self.batch, drift_type='feature')
```

Comment:
See `predict` function signature for univariate drift detectors for reference [here](https://github.com/SeldonIO/alibi-detect/blob/master/alibi_detect/cd/base.py#L892)